### PR TITLE
GP2 135 user activated account yes or no

### PR DESCRIPTION
**Pull request by markcsims**
- adds activated date field to user
- sets activatedDate when user clicks on invitation
- activated date is updated in controller
- returns activated date from squidex
- fixes activedDate in schema
- updates schema

### CODE REVIEW

File: apps/gp2-frontend/src/users/export.ts [hunk 1]
```diff
@@ -20,7 +20,8 @@ export const userFields = {
   workingGroups: 'Working groups',
   fundingStreams: 'Funding streams',
   contributingCohorts: 'Contributing cohorts',
-  createdDate: 'Activated account',
+  createdDate: 'Account Created',
+  activatedDate: 'Activated account',
```
Comment (jhonny-m) [inline apps/gp2-frontend/src/users/export.ts:24]: shouldn't it be Account Activated?

File: apps/gp2-server/src/controllers/user.controller.ts [hunk 2]
```diff
@@ -96,6 +96,7 @@ export default class Users implements UserController {
       {
         email: user.email,
         connections: [...(user.connections || []), { code: authUserId }],
+        activatedDate: user.activatedDate ?? new Date().toISOString(),
```
Comment (jhonny-m) [inline apps/gp2-server/src/controllers/user.controller.ts:99]: why do we default it to a new Date?

Comment (jhonny-m) [inline apps/gp2-server/src/controllers/user.controller.ts:99]: whats the scenario where we would have a date already?

**Review by jhonny-m — APPROVED** [review]

File: apps/gp2-frontend/src/users/export.ts (hunk 1 above)
Comment (markcsims) [reply apps/gp2-frontend/src/users/export.ts:24]: It's what we had, but I think it should change.

File: apps/gp2-server/src/controllers/user.controller.ts (hunk 2 above)
Comment (markcsims) [reply apps/gp2-server/src/controllers/user.controller.ts:99]: There is. All goes back to the conversations we're had in RTM about connections. 

I think a user will be allow multiple connection, so could go past this code more that once. 

If that happens, I think we should keep the first activated date.

**Pull request merged**